R/subscript-tables.R

```diff
@@ -147,7 +155,29 @@ updateTableAttributes <- function(y, x, called.args) {
     attr.names <- names(attributes(y))
     names.needing.update <- !isBasicAttribute(attr.names)
     names(attributes(y))[names.needing.update] <- paste0("original.", attr.names[names.needing.update])
+    y <- updateSpanIfNecessary(y, x.attributes, called.args, evaluated.args)
     attr(y, "name") <- paste0(x.attributes[["name"]], "[",
                               paste(as.character(called.args), collapse = ","), "]")
     y
 }
+
+subscriptSpanDF <- function(span.attr, idx) {
+    if (is.character(idx))
```

Comment:
Add empty span check (only one column in span df) and return empty early

Reply:
Resolved in https://github.com/Displayr/verbs/pull/43/commits/aaba8b4a9f1a0a07f9e7c48b88b9b5771903fc3a